pkgs/by-name/sn/sncli/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  python312Packages,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  __Simperium3 = python312Packages.buildPythonPackage rec {
+    pname = "Simperium3";
+    version = "0.1.5";
+
+    src = python312Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "0dyi2h0s3673kngns3rx8x7sv51bxi8dv0syv1gsssc8w5miif3q";
+    };
+
+    propagatedBuildInputs = with python312Packages; [ requests ];
+  };
+in
+python312Packages.buildPythonPackage rec {
+  pname = "sncli";
+  version = "0.4.2";
+  pyproject = true;
+  build-system = [ python312Packages.setuptools ];
+
+  src = pkgs.fetchFromGitHub {
+    owner = "insanum";
+    repo = "sncli";
+    rev = "befe46abf978e315472ade27e841c0a21fe6b467";
+    sha256 = "sha256-Ldm8oQdJvZXjD2ZdnkK+HZjMkbDXGkJSkai3iuhNziw";
+  };
+
+  propagatedBuildInputs = with python312Packages; [
+    urwid
+    requests
+    __Simperium3
+    setuptools
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/insanum/sncli";
+    maintainers = with maintainers; [ congee ];
```

Comment:
Please add yourself to `maintainers/maintainer-list.nix` in a separate commit and use your name here.